pkg/cloud/amazon/common_test.go

```diff
@@ -2,34 +2,43 @@ package amazon_test
 
 import (
 	"github.com/jenkins-x/jx/pkg/cloud/amazon"
-	"github.com/pborman/uuid"
 	"github.com/stretchr/testify/assert"
 	"io/ioutil"
 	"os"
 	"path"
+	"runtime"
 	"testing"
 )
 
 func switchHome() (string, error) {
-	oldHome := os.Getenv("HOME")
-	newHome := "/tmp/" + uuid.New()
-	os.Setenv("HOME", newHome)
+	oldHome := amazon.UserHomeDir()
+	newHome, err := ioutil.TempDir("", "common_test")
```

Comment:
BTW - this doesn't look like it gets cleaned up ever....

Reply:
No, it doesn't. But it's only in the temp directory and the files are only small, so not a biggie.